### PR TITLE
Wrap external calls with try-except

### PR DESCRIPTION
**Pull request by andela-mmakinde**
Solves https://github.com/CodeForAfricaLabs/HealthTools.API/issues/53

Any background context you want to provide?
- None

Screenshots (if appropriate)
screen shot 2017-11-08 at 10 30 42 am
screen shot 2017-11-08 at 10 34 22 am

### CODE REVIEW

File: healthtools/core.py [hunk 1]
```diff
@@ -63,3 +63,14 @@ def __call__(self, req):
...
+def print_error(message):
+    '''
+    Print error messages in the terminal.
+    '''
+    error = "- ERROR: " + message['ERROR']
+    source = ("- SOURCE: " + message['SOURCE']) if "SOURCE" in message else ""
+    error_msg = "- MESSAGE: " + message['MESSAGE']
+    msg = "\n".join([error, source, error_msg])
+
+    log.error(msg)
```
Comment (DavidLemayian) [inline healthtools/core.py]: Probably able to be achieved with formatting? - https://docs.python.org/3/howto/logging-cookbook.html

**Comment by DavidLemayian** [comment]
Also why did you move `healthtools/manage.py → manage.py`?

**Comment by andela-mmakinde** [comment]
manage.py is meant to be in the root folder. Currently in the develop branch, the file is in the healthtools folder and the application doesn't run except the file is placed in the root.

File: healthtools/core.py (hunk 1 above)
Comment (andela-mmakinde) [reply healthtools/core.py]: Trying it out.

**Pull request merged**